src/libraries/Common/src/System/Security/Cryptography/MLDsaImplementation.OpenSsl.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Security.Cryptography
+{
+    internal sealed partial class MLDsaImplementation : MLDsa
```

Comment:
The OpenSSL implementation won't need to be shared in Microsoft.Bcl.Cryptography; so it can go straight into src/libraries/src/System.Security.Cryptography/src/System/Security/Cryptography